std/format/internal/write.d

```diff
@@ -28,15 +28,19 @@ package(std.format):
     `bool`s are formatted as `"true"` or `"false"` with `%s` and as `1` or
     `0` with integral-specific format specs.
  */
-void formatValueImpl(Writer, T, Char)(auto ref Writer w, T obj, scope const ref FormatSpec!Char f)
+void formatValueImpl(Writer, T, Char)(auto ref Writer w, const(T) obj, scope const ref FormatSpec!Char f) // const(T) reduces number of instances
 if (is(BooleanTypeOf!T) && !is(T == enum) && !hasToString!(T, Char))
 {
-    BooleanTypeOf!T val = obj;
+    BooleanTypeOf!T val = obj;  // TODO: remove and pass obj to val directly in line below?
+    formatValueImplBool!(Writer, Char)(w, val, f);
+}
 
+void formatValueImplBool(Writer, Char)(auto ref Writer w, const(bool) val, scope const ref FormatSpec!Char f)
+{
     if (f.spec == 's')
         writeAligned(w, val ? "true" : "false", f);
     else
-        formatValueImpl(w, cast(byte) val, f);
+        formatValueImplIntegral(w, cast(const(byte))val, f);
```

Comment:
The style checker complains about such casts. A space should be between the cast and the value that is being cast `cast(type) val`.

Reply:
This also applies to code in comments...

Reply:
Fixed.